File: SPLN/aulas_kiko/aula2/ex2.py
#!/usr/bin/python3

#Função que recebe lista de números e imprime pares

# Lista
lista = [1,2,3,4,5,6]

# Funçoes
## Normal
def paresNormalPrint (listaT) :
    for item in listaT:
        if item%2==0:
            print (item)
print("Normal:")
paresNormalPrint(lista)

## Oneline (funcao)
def paresOnelineFunction (listaT):
    [print(i) for i in listaT if i%2==0]
print("Oneline Function:")
paresOnelineFunction(lista)

## Oneline (em compreensao)
print("Oneline:")
print([i for i in lista if i%2==0])

File: SPLN/aulas_kiko/aula1 manuel node/ex2.js
#!/usr/bin/env node

const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];

for (let i = 0; i < array.length; i++) {
	if (!(array[i] % 2)) {
		console.log('Par: ' + array[i]);
	}
}
